vaadin-renderer-flow-parent/vaadin-renderer-flow/src/main/resources/META-INF/resources/frontend/flow-component-directive.js

```diff
@@ -11,22 +11,28 @@ class FlowComponentDirective extends Directive {
   }
 
   update(part, [appid, nodeid]) {
+    this.updateContent(part, appid, nodeid);
+    return noChange;
+  }
+
+  updateContent(part, appid, nodeid) {
     const { parentNode, startNode } = part;
 
     const newNode = this.getNewNode(appid, nodeid);
     const oldNode = this.getOldNode(part);
 
-    if (oldNode === newNode) {
-      return noChange;
+    if (!newNode) {
+      // If the node is not found, try again later.
+      setTimeout(() => this.updateContent(part, appid, nodeid));
+    } else if (oldNode === newNode) {
+      return;
     } else if (oldNode && newNode) {
       parentNode.replaceChild(newNode, oldNode);
     } else if (oldNode) {
       parentNode.removeChild(oldNode);
```

Comment:
Once we have `if (!newNode)`, this branch will never be executed. I think we can remove it.

Reply:
Removed